README.md

```diff
@@ -1,3 +1,5 @@
 [Pyromaths](http://pyromaths.org/) est un programme qui a pour but de créer des exercices type de mathématiques niveau collège et lycée ainsi que leur corrigé. C'est ce qu'on appelle parfois un exerciseur. Contrairement à de nombreux autres projets, Pyromaths a pour objectif de proposer une correction véritablement détaillée des exercices proposés et pas seulement une solution.
 
 Il permet par exemple de proposer des devoirs maison aux élèves et de leur distribuer ensuite la correction. Il peut aussi servir à des familles afin qu'un élève puisse travailler un point du programme et se corriger ensuite.
+
+Si vous voulez aider avec la traduction, consultez [cette page](https://github.com/Pyromaths/pyromaths/blob/develop/locale/TRADUIRE.md).
```

Comment:
I really appreciate your help. All my french was at school. I do not quite understand the semantics well.